src/neuroconv/datainterfaces/ecephys/neuroscope/neuroscope_utils.py

```diff
@@ -3,8 +3,6 @@
 
 from dateutil import parser
 
-from ....tools import get_package
-
```

Comment:
recommend re-including

Reply:
No problems with that, whatever works best I just wanted to have the dev tests passing.